packages/venia-concept/src/RootComponents/Category/category.js

```diff
@@ -57,6 +57,13 @@ class Category extends Component {
         id: 3
     };
 
+    componentDidUpdate(prevProps) {
+        // If the current page has changed, scroll back up to the top.
+        if (this.props.currentPage !== prevProps.currentPage) {
+            window.scrollTo(0, 0);
+        }
+    }
```

Comment:
Any time props change, scroll to top? Could that affect more cases than just the intended one?

Reply:
Only happens when the `currentPage` prop changes, not when any prop changes.

In english, only when the page changes do we scroll to the top. I think that's exactly what we want, no more / no less.